tests/rptest/services/templates/redpanda.yaml

```diff
@@ -63,8 +63,12 @@ pandaproxy:
     authentication_method: {{ pandaproxy_config.authn_method }}
     {% endif %}
 
+  {% if pandaproxy_config.cache_keep_alive_ms %}
```

Comment:
nit: `Optional[int]` really means `None or int` so just to be pedantic you might want to do `if ..._alive_ms is not None` because otherwise values of `0` also evaluate to false.